test/common/scale_up_replicas_threescale.go

```diff
@@ -0,0 +1,152 @@
+package common
+
+import (
+	goctx "context"
+	"fmt"
+	"testing"
+	"time"
+
+	threescalev1 "github.com/3scale/3scale-operator/pkg/apis/apps/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	numberOfReplicas int64 = 2
+	scaleUpReplicas  int64 = 3
+	name                   = "3scale"
+	namespace              = "redhat-rhmi-3scale"
+)
+
+func TestReplicasInThreescale(t *testing.T, ctx *TestingContext) {
+
+	createWorkspaceRetryInterval := time.Second * 20
+	createWorkspaceTimeout := time.Minute * 7
+
+	apim, err := getAPIManager(ctx.Client, t)
+	if err != nil {
+		t.Fatalf("failed to get APIManager : %v", err)
+	}
+
+	t.Log("Checking correct number of replicas are set")
+	if err := checkNumberOfReplicasAgainstValue(apim, numberOfReplicas, createWorkspaceRetryInterval, createWorkspaceTimeout, t); err != nil {
+		t.Fatalf("Incorrect number of replicas to start : %v", err)
+	}
+
+	apim, err = updateAPIManager(ctx.Client, t, scaleUpReplicas)
+	if err != nil {
+		t.Fatalf("Unable to update : %v", err)
+	}
+
+	t.Log("Checking correct number of updated replicas are set")
+	if err := checkNumberOfReplicasAgainstValue(apim, scaleUpReplicas, createWorkspaceRetryInterval, createWorkspaceTimeout, t); err != nil {
+		t.Fatalf("Incorrect number of replicas : %v", err)
+	}
+
+	apim, err = updateAPIManager(ctx.Client, t, numberOfReplicas)
```

Comment:
This should be passing in a lower numberOfReplicas such as 1. To ensure the operator changes it back to 2.